tests/test_material.py

```diff
@@ -1290,7 +1290,47 @@ def test_mcnp_mat0():
     assert_equal(mass, mass_exp)
 
 
+def test_uwuw():
+    leu = Material(nucvec={'U235': 0.04, 'U238': 0.96},
+                   metadata={'mat_number': 2,
+                          'table_ids': {'92235':'15c', '92238':'25c'},
+                          'mat_name':'LEU',
```

Comment:
Help me out here. Why both `mat_name` and `name` in the material def?

Reply:
this is actually a good question, I copy pasted the one form `mcnp()` test... I should probably fix both of them !

Reply:
looking at the source code of material.cpp, it seems that openmc is using the `mat_name` metadata to name the material where all the others uses `name`.

Here is what I suggest:
In this PR: fix my test about `uwuw` removing mat_name
Issue an issue: to remove mention to mat_name in `openmc()` and in the test_material.py 
@pshriwise what do you thing ?

Shall we also implement something to catch old `mat_name` and convert them into name ?

Reply:
That sounds like a good plan to me!